Project/app/src/main/java/org/ionproject/android/common/ionwebapi/IonService.kt

```diff
@@ -7,7 +7,7 @@ import retrofit2.http.Url
 
 interface IonService {
 
-    @Headers("Authorization: Bearer $WEB_API_AUTHORIZATION_TOKEN")
+    //@Headers("Authorization: Bearer $WEB_API_AUTHORIZATION_TOKEN")
```

Comment:
Remove this. We should not have commented out code in the code base. If it is not necessary, remove it. If it his important to keep it commented out, add information to explain why.